packages/blockchain-wallet-v4-frontend/src/modals/Mobile/MobileNumberAdd/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { bindActionCreators, compose } from 'redux'
+
+import { actions } from 'data'
+import modalEnhancer from 'providers/ModalEnhancer'
+
+import { getData } from './selectors'
+import MobileNumberAdd from './template.js'
+
+class MobileNumberAddContainer extends React.PureComponent {
+  constructor(props) {
+    super(props)
+    this.onSubmit = this.onSubmit.bind(this)
```

Comment:
if you use arrow function then you do not need this one.

Reply:
For sure we should! But for now, it is just what I did for the linter